torch_tvm/compiler.cpp

```diff
@@ -34,6 +36,14 @@ std::vector<DLManagedTensorPtr> set_input(
     //auto tensor = ivalue.toTensor().to(at::kFloat);
     auto tensor = ivalue.toTensor();
     DLManagedTensor* dl_tensor;
+#ifdef TVM_TRAINING
```

Comment:
can this logic be merged below and throw a runtime error the tensor is unaligned/non-contiguous and `TVM_TRAINING` is defined?

Reply:
Actually I don't understand why this is a static flag either and why training requires this. Is this more of a CPU vs CUDA thing?

Reply:
I updated it to a runtime flag in the latest update